libs/feature/dataviz/src/lib/geo-table-view/geo-table-view.component.spec.ts

```diff
@@ -12,7 +12,7 @@ import {
   MapManagerService,
 } from '@geonetwork-ui/feature/map'
 import { FEATURE_COLLECTION_POINT_FIXTURE_4326 } from '@geonetwork-ui/common/fixtures'
-import { Map } from 'ol'
+import { Feature, Map } from 'ol'
```

Comment:
Same as above, it would be great if you could turn this into more precise imports!